src/custom-elements/src/components/time/time.tsx

```diff
@@ -188,12 +222,12 @@ export class Time {
     const hoursToDisplay = [
       <div data-hour={startHour} class={`
         ${TIME_CLASSES.HOUR}
-        ${hourStatus(startHour.toString())}
+        ${hourStatus(startHour)}
         `}
-           onClick={() => setHour(startHour.toString())}>{startHour}</div>
+           onClick={() => setHour(hourFormat.toString())}>{startHour}</div>
```

Comment:
I would use this kind of casting instead of toString(), since this one isn't guaranteed to return a string
```suggestion
           onClick={() => setHour(String(hourFormat))}>{startHour}</div>
```

Reply:
Updated 😉